src/tox/_pytestplugin.py

```diff
@@ -107,12 +108,18 @@ def __init__(self, capfd, args):
 
     def __enter__(self):
         self._start = time.time()
-        self._capfd.readouterr()
+        self._out_tell = sys.stdout.tell()
+        self._err_tell = sys.stderr.tell()
```

Comment:
hmmm how does this work

```
>>> pos = sys.stdout.tell()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
io.UnsupportedOperation: underlying stream is not seekable
```

Reply:
it will not if you do against stdin, but it does against pytest replaced stdin, where this tests do run 👍

Reply:
Only by default, `pytest -s` and 💥

Reply:
Any sane way to get the run capfd from a fixture? (yeah this was a workaround for that)

Reply:
does the original way (in master) work?

Reply:
only if you passed on the capfd from the test function, but I would like to avoid having to pass through that every time, as it's only used by this underlying fixture to collect the runtime output 👍